.github/workflows/issues-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+name: Add issues to project
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.5.0
+        with:
+          project-url: https://github.com/orgs/Kuadrant/projects/5
+          github-token: ${{ secrets.ADD_ISSUES_TOKEN }}
```

Comment:
where does this secret come from?

Reply:
@guicassolato created the PAT targeting the kuadrant organisation, specifically the five repos that the PR's are open for